static/templates/admin_tab.handlebars

```diff
@@ -158,6 +159,28 @@
               </div>
             </form>
           </div>
+          <div id="organization-settings" class="settings-section">
+            <div class="settings-section-title"><i class="icon-vector-gear settings-section-icon"></i>
+                {{t "Authentication Methods" }}</div>
+              <form class="form-horizontal admin-realm-form">
+               <div class="alert" id="admin-realm-authentication-methods-enabled-status"></div>
+                <div class="admin-table-wrapper">
+                  <p>{{#tr this}}Configure the authentication methods for {{domain}} organization.{{/tr}}</p>
```

Comment:
s/for domain organization/for the domain organization/.

Also, with `tr this` I think you need to use `__domain__`, not `{{domain}}` as the translation tagging syntax